ntbs-service/Startup.cs

```diff
@@ -36,7 +43,35 @@ public void ConfigureServices(IServiceCollection services)
             });
 
 
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            IConfigurationSection adfsConfig = Configuration.GetSection("AdfsOptions");
+            var authSetup = services.AddAuthentication(sharedOptions =>
+                    {
+                        sharedOptions.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                        sharedOptions.DefaultSignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                        sharedOptions.DefaultChallengeScheme = WsFederationDefaults.AuthenticationScheme;
+                    }).AddWsFederation(options =>
+                    {
+                        options.MetadataAddress = adfsConfig["AdfsUrl"];
+                        options.Wtrealm = adfsConfig["Wtrealm"];
+                    })
+                    .AddCookie(options =>
```

Comment:
Why do we need this cookie for Dev?

Reply:
What I'm trying to achieve here is mimicking the live behaviour as close as it makes sense and limiting the `DevAuthHandler` to only "do" the authentication, while leaving sign-in / sign-out behaviour up to the cookie handler.
That way there's just one method to implement on `DevAuthHandler`, too.

Does that make sense? At least that's what I'm aiming for, if  I'd not got something right here then do shout!

Reply:
Ok that makes sense - could we move this string into a static method in the authentication namespace though, which is then also called by the DevAuthHandler?

Reply:
You mean the `"Dev"` string, right? sure thing